src/table/table-role/__tests__/stubs.tsx

```diff
@@ -21,25 +25,25 @@ export const idColumn = { header: 'ID', cell: (item: Item) => item.id };
 export const nameColumn = {
   header: (
     <span>
-      Name <button aria-label="Sort by name" tabIndex={0} />
+      Name <Button aria-label="Sort by name" />
```

Comment:
Is this supposed to use the `Button` component? It doesn't seem to be imported and we are not using its props (e.g, `ariaLabel` instead of `aria-label`.

Reply:
No, that is a custom button component defined down below.